### ISSUE TITLE
blurred validation output

### ISSUE THREAD

**Issue by JohnHerry** [lucidrains/e2-tts-pytorch#43]
cards: 4 ,    batch size:   10 ,    lr: 1e-4       
sample audio :  1s - 20s each,  16K audios,   hop_size=200     
mel-extractor:  same as[ HiFiGAN](https://github.com/jik876/hifi-gan)   
text tokenizer:  phoneme tokens    

I have trained to 250,000 steps but still can not get clear mel-spectrogram output. the validation output is blurred and diluteed artifacts,  
![pic](https://github.com/user-attachments/assets/f7eed345-c104-4db7-bce0-ef9571e7ecfd)    

has anybody run into such a situation? what is the reason? is this problem on training data preprocessing? or need  some training parameter adjustment?  wating for suggestions.   thanks.

**Comment by kunit17** [comment]
are the mel_specs normalized for training?
